pom.xml

```diff
@@ -15,35 +15,26 @@
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
 
     <modelVersion>4.0.0</modelVersion>
-
     <groupId>org.eclipse.uprotocol</groupId>
     <artifactId>up-kotlin</artifactId>
     <name>Kotlin Library for uProtocol</name>
     <description>Language specific uProtocol library for building and using UUri, UUID, UAttributes, UTransport, and more
     </description>
-    <version>1.5.7-SNAPSHOT</version>
+    <version>0.1.8-SNAPSHOT</version>
     <packaging>jar</packaging>
     <url>https://github.com/eclipse-uprotocol/up-kotlin/</url>
 
     <properties>
         <kotlin.version>1.9.21</kotlin.version>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-        <maven.compiler.source>11</maven.compiler.source>
-        <maven.compiler.target>11</maven.compiler.target>
         <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
-
-
         <junit.platform.version>1.9.1</junit.platform.version>
         <junit-jupiter.version>5.9.1</junit-jupiter.version>
         <junit.version>4.12</junit.version>
         <maven-surefire-plugin.version>3.0.0-M5</maven-surefire-plugin.version>
-
         <slf4j.version>2.6.18</slf4j.version>
-
-        <cloudevents.version>2.4.2</cloudevents.version>
-
         <protobuf.version>3.21.10</protobuf.version>
-        <git.tag.name>uprotocol-core-api-1.5.7</git.tag.name>
+        <git.tag.name>main</git.tag.name>
```

Comment:
@dddj698 please update to use v1.6.0-alpha.2 on this PR or the next one but I'll still approve this Pr if I don't find anything else.

Reply:
it's in next PR